src/addon.js

```diff
@@ -135,6 +135,16 @@ class Addon extends EventEmitter {
     });
   }
 
+  upgradePremium() {
+    return new Promise((resolve, reject) => {
+      self.channel.call({
+        method: 'upgradePremium',
+        success () { resolve() },
+        error (err) { reject(err) }
```

Comment:
Curious why passing resolve and reject here direclty instead of wrapping them in success() and error() doesn't work?

Also you will want to version bump so that the dashboard can depend of this updated version with the new `upgradePremium` call. I was also going to say that we should update the README but since it's very unlikely a 3rd party would use this method we can probably dismiss it from the README for now.